core/vm/contracts.go

```diff
@@ -662,15 +665,33 @@ func (c *numberValidators) RequiredGas(input []byte) uint64 {
 }
 
 func (c *numberValidators) Run(input []byte, caller common.Address, evm *EVM, gas uint64) ([]byte, uint64, error) {
-	blockNumber := evm.Context.BlockNumber
-	validators := evm.Context.Engine.GetValidators(blockNumber, evm.Context.GetHash(blockNumber.Uint64()))
 	gas, err := debitRequiredGas(c, input, gas)
 	if err != nil {
```

Comment:
we could check `len(input)!=0` here too

Reply:
I think we want len(input) to equal 0 here